src/services/AdministrationService.js

```diff
@@ -0,0 +1,27 @@
+import { loadBvvAdministration } from './provider/administration.provider';
+import { isCoordinate } from '../utils/checks';
+
```

Comment:
We need a type definition for `Administration` to document the domain object of this service